### PR TITLE
Experimental/sass settings

### PR DESCRIPTION
**Pull request by Kirill9m**
I would like make an suggestion:
While the project is still in the development stage could we switch sass back to the expanded compilation format?
It means what that you can see the original CSSfile in a normal format. 
In this case after each change I won’t end up with multiple files to commit and it will also help avoid merge conflicts.

I did some research, and it seems that during the development stage, the expanded compilation format is more commonly used. 

I also noticed that the compiler had a probem with @import directive in global variabel which caused an error.

What do you all think about this?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
     "description": "A project to compile SCSS to CSS using node-sass or dart-sass",
     "main": "index.js",
     "scripts": {
-        "build-css": "sass styles/layouts/scss:styles/layouts/css --style compressed --source-map",
-        "watch-css": "sass --watch styles/layouts/scss:styles/layouts/css --style compressed --source-map"
+        "build-css": "sass styles/layouts/scss:styles/layouts/css --style expanded --source-map",
+        "watch-css": "sass --watch styles/layouts/scss:styles/layouts/css --style expanded --source-map"
```
Comment (Kirill9m) [inline package.json:8]: .

File: styles/layouts/scss/header.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @use '../../global-styles/_colors.scss' as *;
 @use '../../global-styles/_fonts.scss' as *;
 @use '../../global-styles/_screens.scss' as *;
-@import '../../global-styles/_key-frames.scss';
+@use '../../global-styles/_key-frames.scss' as *;
```
Comment (Kirill9m) [inline styles/layouts/scss/header.scss:5]: .

**Comment by MagnusTerak** [comment]
I agree with you on this, a great way to try avoiding conflicts, and make it less of a "mess" when pushing. 
Also great find about the import compiling error! 💯

**Comment by El-Duckster** [comment]
@Kirill9m @MagnusTerak  ,just to make sure :) We should not directly modify the CSS files, as they are generated from the SCSS files during the Sass compilation process. Instead, all edits should be made in the SCSS files to maintain consistency and avoid overwriting changes. Since we are working on separate parts of the project, the chances of conflicts are minimal. If you need to inspect the CSS for any reason, of course you could use expanded compilation format temporarily for better readability :)

File: styles/layouts/scss/header.scss (hunk 2 above)
Comment (El-Duckster) [reply styles/layouts/scss/header.scss:5]: oops the animation worked fine in the project , this is why i haven't noticed it. how did you detect it ? :)

**Comment by Fagerdahl** [comment]
The compiled CSS in an expanded compilation format is obviously much easier to read. 
How big are the risks of merging conflicts? 
I think a "vite build" could be the answer to your question in this! :-)

File: styles/layouts/scss/header.scss (hunk 2 above)
Comment (Kirill9m) [reply styles/layouts/scss/header.scss:5]: I have just noticed a log in the Sass compiler console :)

**Comment by Kirill9m** [comment]
It were just thoughts 
It doesn't make much of a difference, but maybe Elena is right
if desired this can be done locally as well.
We'll leave everything as it is
PS:
It seems like everyone has already fixed the bug locally in their projects :D so I am closing pr now!

**Comment by Kirill9m** [comment]
> The compiled CSS in an expanded compilation format is obviously much easier to read. How big are the risks of merging conflicts? I think a "vite build" could be the answer to your question in this! :-)

Lets not take any risks :D We can try Vite instead!

**Pull request closed without merging**